app/helpers/session.rb

```diff
@@ -0,0 +1,46 @@
+def authorized?
```

Comment:
This is a global function which is a bad practice. All helpers should be wrapped in 
```ruby
helpers do
  …
end
```